README.md

```diff
@@ -8,29 +8,11 @@ the [Documentation on drupal.org](https://www.drupal.org/node/2471553).
 
 ## Usage
 
-First you need to install [Composer](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-osx) and [Git](https://git-scm.com).
+See our [install documentation](https://thunder.github.io/thunder-documentation/quick-install) on how to use Thunder project.
 
-> Note: The instructions below refer to the [global composer installation](https://getcomposer.org/doc/00-intro.md#globally).
-You might need to replace `composer` with `php composer.phar` (or similar) 
-for your setup.
+We also have documentation on [extending](https://thunder.github.io/thunder-documentation/extending) and [updating](https://thunder.github.io/thunder-documentation/updating) Thunder sites. 
+ Note: The instructions below refer to the [global composer installation](https://getcomposer.org/doc/00-intro.md#globally). 
```

Comment:
I think we should have the same documentation here as for `thunder/thunder-project`. And there we have removed this `Note` and also part with `If you want to know how to use it as replacement for ...`.